### PR TITLE
fix: updated values in apb.yml

### PR DESCRIPTION
**Pull request by psturc**

### CODE REVIEW

File: apb.yml [hunk 1]
```diff
@@ -53,37 +53,37 @@ plans:
...
         title: Sync App Port
       - name: SYNC_APP_GRAPHQL_ENDPOINT
-        description: GraphQL Endpoint of your Sync Application (using 'graphql' if not specified)
+        description: (default value: 'graphql') GraphQL Endpoint of your Sync Application 
         type: string
         required: False
         title: Sync App GraphQL Endpoint
       - name: DB_HOSTNAME
-        description: Hostname of the SQL Server
+        description: Hostname of your Database Server
```
Comment (wei-lee) [inline apb.yml]: maybe just say `Hostname of the Database Server`? If we say `your Database` here, then let's keep using the same in the descriptions for the configurations below.

File: apb.yml [hunk 2]
```diff
@@ -53,37 +53,37 @@ plans:
...
+        title: Database Server Username
       - name: DB_PASSWORD
         description: Password that will be used to connect to Database
         type: string
         required: True
-        title: SQL Server Password
+        title: Database Server Password
       - name: DB_NAME
-        description: Name of the SQL Server's database
+        description: Name of the DB Server's database
```
Comment (wei-lee) [inline apb.yml]: either `you Database` or `the Database`

File: apb.yml [hunk 3]
```diff
@@ -53,37 +53,37 @@ plans:
...
         type: string
         required: True
         title: Database name
       - name: DB_SSL
-        description: Use SSL connection to SQL Server
+        description: Use SSL connection to Database Server
```
Comment (wei-lee) [inline apb.yml]: same here

File: apb.yml [hunk 4]
```diff
@@ -27,22 +27,22 @@ plans:
...
+        description: URL of your GraphQL Server Docker Image, e.g. docker.io/aerogear/voyager-server-example-task
         type: string
         required: True
         title: Sync Application Docker image
       - name: SYNC_APP_DOCKER_IMAGE_VERSION
-        description: Version of your Sync Application Docker Image (using 'latest' if not specified)
+        description: (default value: 'latest') Version of your Sync Application Docker Image
         type: string
         required: False
         title: Sync Application Docker image version
```
Comment (wei-lee) [inline apb.yml]: should there be a default value here?

**Comment by psturc** [comment]
@wei-lee are you ok with merging this guy?

File: apb.yml [hunk 5]
```diff
@@ -27,63 +27,64 @@ plans:
...
+        default: 'latest'
       - name: SYNC_APP_HEALTH_ENDPOINT
-        description: Health Endpoint of your Sync Application (using '/' if not specified)
+        description: (default value: '/') Health Endpoint of your Sync Application. Use 'health' for docker.io/aerogear/voyager-server-example-task image.
```
Comment (wei-lee) [inline apb.yml]: there is already a `/` in the template, should the default value here just be an empty string?

Comment (psturc) [reply apb.yml]: As we discussed, we should also give user an option to not use the health endpoint at all and this is not possible atm (it requires some changes to provision role). I will address this in other PR.

**Pull request merged**